routes/api/users.js

```diff
@@ -27,6 +30,20 @@ router.get('/', async (req,res)=>{
     });
 });
 
+//endpoint para buscar user por su id
+router.get('/:userId',isAdmin, async (req,res)=>{
+    if (isNaN(req.params.userId)) {
+        return responses.errorDTOResponse(res, 400, "El ID debe ser un número");
+    }
+    const user1= await User.findOne({
+        where:{id: req.params.userId}
+    });
+     if(!user1){
+        return responses.errorDTOResponse(res, 404, "No existe usuario con este id");
+    }
+    return responses.singleDTOResponse(res,200,"Recuperación exitosa",user1);
+  });

```

Comment:
```suggestion
    const user = await User.findOne({
        where:{id: req.params.userId}
    });
     if(!user){
        return responses.errorDTOResponse(res, 404, "No existe usuario con este id");
    }
    return responses.singleDTOResponse(res,200,"Recuperación exitosa",user);
  });
```
Chequeen si eso furula así, no me parece que se llame user1 xd